config.ini.example

```diff
@@ -6,6 +6,7 @@ quiet = no
 debug = yes
 
 ## Display specific
+display_class = display
```

Comment:
This option is named `display`, see [`run,py`, line 45](https://github.com/Kartoffel/infodisplay/pull/3/commits/962e020cf0729cd311cbfb2087d729856990292b#diff-d6af0459a37d985953d7040c14f53feb3b9cc9e58b543aa3c2b80256d276c5e0R45).